.gitignore

```diff
@@ -9,3 +9,5 @@ docker/Dockerfile.worker
 inst/output/malawi_output.rds
 inst/output/malawi_spectrum_download.zip
 inst/output/malawi_summary_download.zip
+id_rsa
+tests/testthat/testdata/sensitive
```

Comment:
In order to get the tests to pass we need to do this more carefully (there might be an option [here](https://cran.r-project.org/doc/manuals/R-ints.html#Tools) to disable this check but I do not see it)

1. Add `tests/testthat/testdata/sensitive` into `.Rbuildignore` so that is not built into the package or download the data elsewhere on travis's machine
2. Add a new environment variable `NAOMI_SENSITIVE_DATA_PATH= $TRAVIS_BUILD_DIR/tests/testthat/testdata/sensitive` (goes added into the current env set, all in one ugly line)
3. Your switch then becomes something like
```r
skip_if_sensitive_data_missing <- function() {
  sensitive_path <- Sys.getenv("NAOMI_SENSITIVE_DATA_PATH", "testdata/sensitive")
  if (!file.exists(sensitive_path)) {
    skip("Sensitive data missing, check README for details.")
  }
}
```

That should enable the tests to find the data even though it's not built into the package, but still be a pleasant structure to work with locally

Reply:
Fixed this